docs/examples/use_cases/tensorflow/efficientdet/utils.py

```diff
@@ -30,6 +28,82 @@
 # pylint: disable=logging-format-interpolation
 
 
+class PipelineType(Enum):
+    syntetic = 0
+    tensorflow = 1
+    dali_cpu = 2
+    dali_gpu = 3
+
+
+def dict_to_namedtuple(字典):
+    NamedTuple = namedtuple("NamedTuple", 字典.keys())
+    return NamedTuple._make(字典.values())
```

Comment:
I would prefer to keep the variable names in English and limited to plain ascii. 
```suggestion
def dict_to_namedtuple(dict_instance):
    NamedTuple = namedtuple("NamedTuple", dict_instance.keys())
    return NamedTuple._make(dict_instance.values())
```